pyproject.toml

```diff
@@ -119,3 +119,7 @@ pythonpath = "."
 env = [
   "OPTINIST_DIR=studio/test_data",
 ]
+
+[tool.codespell]
+exclude = '^frontend/(build|node_modules)/|.*\.lock$'
```

Comment:
this rule may be invalid.
- the keyword is not `exclude` but `skip`.
- multiple rules are separated by comma.

So, the rule would be
```
skip = "./frontend/build,./frontend/node_modules,./docs/_build,*.lock"
```
In addition to updating the rule, pls add following extensions.
- *.log
- *.egg-info

Reply:
FYI
you can check your rule is valid by running
```
codespell
```
(If you have not installed, run `pip install codespell`)